lib/mock-axios-types.ts

```diff
@@ -189,6 +191,14 @@ export interface AxiosMockAPI {
      */
     reset: () => void;
 
+    /**
+     * Set a request handler that gets invoked every time a new request comes in
+     *   The handler is invoked with the new request item
+     * 
+     * @param handler the function to invoke with the new request item every time a new request comes in
+     */
+    requestHandler: (handler: RequestHandler) => void;
```

Comment:
I think this makes it more clearer that this is a function, not a property:

```suggestion
    useRequestHandler: (handler: RequestHandler) => void;
```